air/src/trace/snapshots/miden_air__trace__tests__ace_chiplet_layout.snap

```diff
@@ -0,0 +1,35 @@
+---
+source: air/src/trace/tests.rs
+expression: layout
+---
+ACE CHIPLET LAYOUT
+=====================
+
+Chiplet Selectors:
+  Number of ACE selectors: 4
+
+Column Indices:
+  - Selector start index: 0
+  - Selector block index: 1
+  - Context index: 2
+  - Pointer index: 3
+  - Clock index: 4
+  - Eval operation index: 5
+  - ID 0 index: 6
+  - ID 1 index: 9
+  - ID 2 index: 12
```

Comment:
It would be a bit clearer if we ordered the columns by index, to show the overlap depending on the current block.